docs/sample_log_config.yaml

```diff
@@ -38,6 +49,6 @@ loggers:
 
 root:
     level: INFO
-    handlers: [file, console]
+    handlers: [buffer, console]
```

Comment:
surely we shouldn't log to both buffer and console by default? let's get rid of console here?